src/consts/Stages.ts

```diff
@@ -3,8 +3,8 @@ import { RPARequest } from "api/requestsApi";
 interface STARTSTAGE {
   key: string;
   text: string;
-  next: string;
-  nextEventTitle: string;
+  next: (request: RPARequest | undefined) => string;
+  nextEventTitle: (request: RPARequest | undefined) => string;
```

Comment:
Just make it an optional parameter?
```suggestion
  next: (request?: RPARequest) => string;
  nextEventTitle?: (request: RPARequest) => string;
```